network/topics/metrics.go

```diff
@@ -24,6 +24,12 @@ var (
 		Name: "ssv:p2p:pubsub:score:inspect",
 		Help: "Gauge for negative peer scores",
 	}, []string{"pid"})
+
+	// score.go invalidMessageDeliveries value per topic
```

Comment:
I think we can remove this. `score.go` looks like a file from libp2p but it can be moved in a next version and there may be several "score.go"'s there. It's easy to follow what's passed to `metricPubSubPeerP4Score`